spec/controllers/selfservice/advisers_controller/create_spec.rb

```diff
@@ -23,7 +23,7 @@ module SelfService
         end
 
         it 'adds a success flash message' do
-          expect(flash[:notice]).to eq('Saved successfully')
+          expect(flash[:notice]).to eq(I18n.t('self_service.adviser_edit.saved'))
```

Comment:
IIRC the shorter `t` version is a helper in the views and thus doesn't need a prefix. Elsewhere we need to call the method of the same name on `I18n` itself. Certainly, I just took off the prefix in this text and re-ran the test, it failed, couldn't resolve `t`, `NoMethodError`.